src/utilities/util_dump.c

```diff
@@ -144,10 +167,14 @@ util_dump(WT_SESSION *session, int argc, char *argv[])
     /* Open any optional output file. */
     if (ofile == NULL)
         fp = stdout;
-    else if ((fp = fopen(ofile, "w")) == NULL)
+    else if (explore) {
+        fprintf(stderr, "%s: the options are -e and -f are incompatible\n", progname);
```

Comment:
Typo for the 1st "are"?

Reply:
Fixed 1da526bb059bda937ae95bfaac45ede554db16d2